lisa/base_tools/wget.py

```diff
@@ -31,6 +33,7 @@ def install(self) -> bool:
         posix_os.install_packages([self])
         return self._check_exists()
 
+    @retry(LisaException, tries=5, delay=2, backoff=1.5)
```

Comment:
it may waste time when encounter a real issue, like the url doesn't exist

Reply:
To address your worry or wasted time: It wastes much more developer time to skip/fail when there's throttling or another network hiccup. Our more complex runs can take an hour or more to set up if we're applying multiple update steps that involve restarts and status checks, it's ok to spend 7-45 seconds retrying if there is throttling or some other network hiccup. They do happen, I wouldn't have added this retry if I hadn't hit this problem previously.
For one test running once, it's less likely you'll hit a problem. It's different when you have thousands of tests running on an NxN matrix of host builds and hardware.

It would help me if you have an actionable suggestion, such as 'could we reduce the retries to 2 tries' or 'can we reduce the backoff to 1.25 instead of 1.5'